configure.ac

```diff
@@ -430,6 +430,8 @@ MINGW=`echo $host_os | grep mingw32  2>/dev/null`
 AM_CONDITIONAL(MINGW, test ! -z "$MINGW" )
 if test ! -z "$MINGW"; then
 	WSOCKLIB="-lws2_32"
+	LDFLAGS="-no-undefined $LDFLAGS"
+	AC_DEFINE(HAVE_GETTIMEOFDAY)
```

Comment:
That define should be a proper check if possible. Don't know much about what works with mingw and what doesn't.

Doesn't matter though, it has been decided autotools will be removed for the next release (see #152 for example). Try to build with cmake :)